packages/paste-core/utilities/flex/src/index.tsx

```diff
@@ -1,9 +1,10 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
 import {ResponsiveValue} from 'styled-system';
-import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {Box, BoxProps, safelySpreadBoxProps} from '@twilio-paste/box';
 import {FlexboxProps, LayoutProps, MarginProps, PaddingProps} from '@twilio-paste/style-props';
 
+export type AsProp = Pick<BoxProps, 'as'>;
```

Comment:
I don't think the Flex package should be the one to export this.  It might make more sense to have this defined in the `style-props` package.

Reply:
Honestly, even that seems like too much. 

I'm leaning towards just declaring as on the interface when ever you need it as `as?: keyof JSX.IntrinsicElements;`

Reply:
Works for me!